commands.js

```diff
@@ -64,6 +64,28 @@ class Commands {
     selfDestructMessage(() => message.reply('added time to user.'))
   }
 
+  async bitrate (message) {
+    let args = message.split(' ').splice(1).filter(str => str !== '')
+    let mem = message.member
+    let channel = message.member.voiceChannel
+    if (channel == null) {
+      throw new Error(`<@${message.author.id}>! You aren't in a voice channel`)
+    }
+    if (args.length === 0) {
+      selfDestructMessage(() => message.reply(`<#${channel.id}> bitrate: ` + channel.bitrate))
+    } else {
+      if (isNaN(args[0])) {
+        throw new Error(`<@${message.author.id}>! Input is not a valid bitrate(i.e. 64000)`)
```

Comment:
Usability: consider telling the user to please enter a number between 8 and 384, and then also check that the argument is within that range. (I think it would be better to let the user specify bitrates in kbps instead of bps.)